admin_manual/installation/example_ubuntu.rst

```diff
@@ -1,23 +1,40 @@
 .. _ubuntu_installation_label:
 
-Example installation on Ubuntu 18.04 LTS
+Example installation on Ubuntu 20.04 LTS
 ========================================
 
-Or you can use .deb packages to install the required and recommended modules for a typical Nextcloud
+You can use .deb packages to install the required and recommended modules for a typical Nextcloud
 installation, using Apache and MariaDB, by issuing the following commands in a
 terminal::
 
-    apt-get install apache2 mariadb-server libapache2-mod-php7.2
-    apt-get install php7.2-gd php7.2-json php7.2-mysql php7.2-curl php7.2-mbstring
-    apt-get install php7.2-intl php-imagick php7.2-xml php7.2-zip
+    apt install apache2 mariadb-server libapache2-mod-php7.4
+    apt install php7.4-gd php7.4-mysql php7.4-curl php7.4-mbstring php7.4-intl
+    apt install php7.4-gmp php7.4-bcmath php-imagick php7.4-xml php7.4-zip
```

Comment:
We should actually just have one apt install command or separate them by required/optional php extensions.

Reply:
Well, I agree... but this matches the original documentation.  I assume it was there for line-length purposes.

Reply:
Yes, I guess so. Fine to keep for now from my side :)